File: secao-12-Linguagem-C++/exercicios/retangulo.cs
// Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos. 

using System;
using System.Globalization;

namespace Retangulo
{
    class Program
    {
        static void Main(string[] args)
        {
            CultureInfo ci = CultureInfo.InvariantCulture;

            double largura, altura, perimetro, area, diagonal;

            Console.Write("Base do retangulo: ");
            largura = double.Parse(Console.ReadLine(), ci);

            Console.Write("Altura do retangulo: ");
            altura = double.Parse(Console.ReadLine(), ci);

            area = largura * altura;
            perimetro = 2 * (altura + largura);
            diagonal = Math.Sqrt(Math.Pow(largura, 2) + Math.Pow(altura, 2)); // Fórmula corrigida

            Console.WriteLine("AREA = " + area.ToString("F4", ci))  ; // Formatação com 4 casas decimais
            Console.WriteLine("PERIMETRO = " + perimetro.ToString("F4", ci)); // Formatação com 4 casas decimais
            Console.WriteLine("DIAGONAL = " + diagonal.ToString("F4", ci)); // Formatação com 4 casas decimais
        }
    }
}

File: secao-13-Linguagem-Python/exercicios/retangulo.py
# Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos.
import math

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = base * altura
perimetro = 2 * (base + altura)
diagonal = math.sqrt(base ** 2 + altura ** 2)

print(f"ALTURA = {area:.4f}")
print(f"PERIMETRO = {perimetro:.4f}")
print(f"ALTURA = {diagonal:.4f}")
